Automattic/newspack-ads
# feat(scaip): opt-out of integrated placements

Allow users to disable the integrated placements for mid-content ad placement and re-enable SCAIP generated sidebars.

### How to test

1. Check-out this branch, https://github.com/Automattic/newspack-plugin/pull/1329 (checkbox fix) and visit the **Advertising -> Settings** wizard.
2. Make sure you have SCAIP plugin installed
2. Observe the new option on the **Post ad inserter settings** section for the use of legacy widgets
3. Confirm it is **enabled** by default
4. Visit the **Placements** wizard and observe that the post insertion placements are not available
5. Visit **Appearance -> Widgets** page and observe the standard SCAIP method is working as expected
6. Now disable the new option and observe the Post insertion placements are working as expected on the **Placements** wizard and the SCAIP widgets are gone
